eng/Versions.props

```diff
@@ -93,6 +93,8 @@
     <MicrosoftBuildFrameworkPackageVersion>15.8.166</MicrosoftBuildFrameworkPackageVersion>
     <MicrosoftBuildLocatorPackageVersion>1.2.6</MicrosoftBuildLocatorPackageVersion>
     <MicrosoftBuildPackageVersion>15.8.166</MicrosoftBuildPackageVersion>
+	  <MicrosoftExtensionsCachingAbstractions>3.1.0</MicrosoftExtensionsCachingAbstractions>
+    <MicrosoftExtensionsCachingMemory>3.1.0</MicrosoftExtensionsCachingMemory>
```

Comment:
@dougbu if we depended on the latest and greatest (5.0.0-x) instead of this pinned version would that introduce a  build time dependency issue?

Reply:
@dougbu  In that case, is there any harm in adding these two packages to the `Label="Automated"` block? Do we need to do any sort of additional registration to ensure they're updated?

Reply:
You'll need to update Version.Details.xml too. Just dupe the four lines in there for Microsoft.Extensions.Configuration.Json then fix the package name.

Reply:
Updated and added thanks.